default/AI/AIstate.py

```diff
@@ -268,6 +273,39 @@ def assess_planet_threat(self, pid, sightingAge=0):
         infra = 0  # doesn't really contribute to combat since damage to shields & 'defense' done first
         return {'overall': defense*(defense + shields + infra), 'attack': defense, 'health': (defense + shields + infra)}
 
+    def log_immobile_fleet(self, fleet, system, rating):
```

Comment:
Single usage of this method is commented out. Are we still need it? 
It can be not instance method but simple function.